Sources/SafeDI/DelayedInstantiation/ErasedInstantiator.swift

```diff
@@ -21,38 +21,34 @@
 /// A SafeDI dependency designed for the deferred instantiation of a type-erased instance of a
 /// type decorated with `@Instantiable`.
 ///
-/// This instantiator can be used to instantiate types that are @MainActor-bound.
-///
 /// - SeeAlso: `Instantiator`
-/// - SeeAlso: `NonisolatedInstantiator`
-/// - SeeAlso: `NonisolatedErasedInstantiator`
+/// - SeeAlso: `SendableErasedInstantiator`
+/// - SeeAlso: `SendableInstantiator`
```

Comment:
IMO people will use the non-`Erased` version more, so move it first like you had before

Reply:
😅 my thinking is that the `SendableErasedInstantiator` is more relevant to the `ErasedInstantiator` type than the `SendableInstantiator`

Reply:
```suggestion
/// - SeeAlso: `SendableInstantiator`
/// - SeeAlso: `SendableErasedInstantiator`
```